lib/realtime/tenants.ex

```diff
@@ -203,7 +203,11 @@ defmodule Realtime.Tenants do
     external_id
     |> Cache.get_tenant_by_external_id()
     |> Tenant.changeset(%{suspend: true})
-    |> Repo.update()
+    |> Repo.update!()
+    |> tap(fn _ ->
+      Cache.invalidate_tenant_cache(external_id)
```

Comment:
what happens if there is a cache on other nodes?

Reply:
it would not be invalidated which is an issue, we would need to trigger this action from the right node which would increase time to ticket resolution

will find another approach

Reply:
pushed a potential approach